scripts/clinic-website-scraper.py

```
@@ -0,0 +1,651 @@
+
+# ── LLM Extraction ────────────────────────────────────────────────────────────
+    combined = "\n\n---PAGE BREAK---\n\n".join(pages_text)
+    content = combined[:MAX_TOTAL_CHARS]
+    merged = {}
+
+    for batch_name, schema_prompt in BATCH_PROMPTS:
+        prompt = f"""{schema_prompt}
+
+Return ONLY the JSON object, nothing else. No explanation. No markdown.
+
+Website: {url}
```

Ruff (0.15.11)

[error] Undefined name `url`

(F821)